charts/k8ssandra/charts/cass-operator/templates/webhook-service.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     name: cass-operator-webhook
 spec:
   ports:
-  - port: 443
-    targetPort: 8443
+    - port: 443
```

Comment:
These type of changes make me want the lint test to check for correct (and defined) yaml indentation. Would make it a lot simpler to read these PRs if each one didn't redo some sort of indenting.